fudy/NoteScreen/StickNoteModel.cpp

```diff
@@ -34,6 +33,12 @@ StickNoteModel::StickNoteModel(QObject* parent)
 	}
 }
 
+using FD = FudyProperties;
```

Comment:
I dont think FudyProperties is a long word. FD take me a while to know it is FDProp

Reply:
I updated this code. Thank you.